dll/win32/shell32/COpenWithMenu.cpp

```diff
@@ -862,6 +884,7 @@ BOOL COpenWithDialog::IsNoOpen(HWND hwnd)
 VOID COpenWithDialog::AddApp(COpenWithList::SApp *pApp, BOOL bSelected)
 {
     LPCWSTR pwszName = m_pAppList->GetName(pApp);
+    if (!pwszName) return ;
```

Comment:
```suggestion
    if (!pwszName) return;
```
and same in the other AddApp function below.